spec/vcr_cassettes/sitemap_indexer_index/when_the_sitemap_is_an_rss_2/0_feed.yml

```diff
@@ -0,0 +1,101 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://www.whitehouse.gov/robots.txt
```

Comment:
This new cassette was recorded due to the indexer's final call to `index_urls`, which hits the domain's robots.txt file to extract the `delay`:
```ruby
  def index_urls
    index!
    SearchgovDomainIndexerJob.perform_later(searchgov_domain: self, delay: delay)
    ...
 ```
 I used a new domain in that spec simply because we had an existing RSS fixture that I could use for testing (`/rss/wh_blog.xml`).

Reply:
Makes sense.  Thanks for the explanation.